### PR TITLE
ci: Add Python Formatting Check to CI

### PR DESCRIPTION
**Pull request by panicstevenson**
Add Python formatting check to Travis CI integration. The Python
formatter of choice is black, since it is unwaivering. For now, to make
pull request readability a thing, string normalization is skipped.

### CODE REVIEW

File: util/test_timeline.py [hunk 1]
```diff
@@ -300,15 +317,23 @@ def check_event(event, timelist, state):
             if not state['timeline_stopped']:
                 for other_entry in timelist:
                     if (
-                            'regex' in other_entry and
-                            other_entry['time'] > state['last_jump'] and
-                            other_entry['time'] < entry['time'] and
-                            other_entry['branch'] < entry['branch']
+                        'regex' in other_entry
+                        and other_entry['time'] > state['last_jump']
```
Comment (quisquous) [inline util/test_timeline.py]: I think this "operator at the start of the line" thing is the most personally distasteful style thing I see in this whole change, but I will take it at the expense of automatic formatting for everything else.

Comment (panicstevenson) [reply util/test_timeline.py]: Yea, some of it is gross.. We can still change courses and divert to something else if you have another preference

Comment (panicstevenson) [reply util/test_timeline.py]: In other news, I'm changing all my if statements from
```python
if x
  and y
```
to
```python
if x:
  if y:
```

Comment (quisquous) [reply util/test_timeline.py]: Yeah, I haven't investigated a ton of other ones.  Did you have anything else in mind, or should I go looking?

**Comment by quisquous** [comment]
I looked at other formatters and I think this is good enough for me.  I think the only thing really missing is a convenient npm command to run black with the right args and reformat when there are errors, along the lines of `npm run lintfix`.

**Pull request merged**